### PR TITLE
Update widgets_UploadInput.php

### PR DESCRIPTION
**Pull request by Felli**
Fix for `es` language breaking.

### CODE REVIEW

File: protected/humhub/modules/file/messages/es/widgets_UploadInput.php [hunk 1]
```diff
@@ -1,22 +1,6 @@
 <?php
-/**
```
Comment (githubjeka) [inline protected/humhub/modules/file/messages/es/widgets_UploadInput.php]: Why is deleted?

Comment (Felli) [reply protected/humhub/modules/file/messages/es/widgets_UploadInput.php]: This is an auto-generated message which isn't really needed and can be removed.

Comment (githubjeka) [reply protected/humhub/modules/file/messages/es/widgets_UploadInput.php]: auto-generated message if the file is new and was created. After removed the message you never see it again. 
That why to need restore it.

Comment (Felli) [reply protected/humhub/modules/file/messages/es/widgets_UploadInput.php]: Shouldn't matter once a pull from the translation site is made, originally the message wasn't included in the translation file. :thinking:

Comment (githubjeka) [reply protected/humhub/modules/file/messages/es/widgets_UploadInput.php]: According to my observations, the bot sends diff changes translation from lasted saved commit. Like `git pull`... Sometimes he pushes incorrect commit and rewrite GitHub translation, if it not was fixed. See https://github.com/humhub/humhub/issues/2679

Comment (githubjeka) [reply protected/humhub/modules/file/messages/es/widgets_UploadInput.php]: If it fixed, Bot will send only new translation row, not this message. If it not fixed,  Bot misses your changes and rewrite like my commit.
In any case, I recommend returning this message.

Comment (Felli) [reply protected/humhub/modules/file/messages/es/widgets_UploadInput.php]: Good point, once I'm on my work laptop I'll fix this. :smile:

Comment (Felli) [reply protected/humhub/modules/file/messages/es/widgets_UploadInput.php]: Should be fixed now. :+1:

**Pull request merged**